test/io/RootImporter.test.cc

```diff
@@ -170,3 +181,54 @@ TEST_F(RootImporterTest, import_geometry)
         i++;
     }
 }
+
+//---------------------------------------------------------------------------//
+TEST_F(RootImporterTest, import_material_params)
+{
+    RootImporter import(root_filename_.c_str());
+    auto         data = import();
+
+    // Material labels
+    std::string material_label;
+    material_label = data.material_params->id_to_label(MaterialDefId{0});
+    EXPECT_EQ(material_label, "G4_Galactic");
+    material_label = data.material_params->id_to_label(MaterialDefId{1});
+    EXPECT_EQ(material_label, "G4_STAINLESS-STEEL");
+
+    auto mat_host_ptr = data.material_params->host_pointers();
+
+    /*!
+     * Material
+     *
+     * Geant4 has outdated constants. The list includes h_planck, na_avogadro,
+     * electron_charge_si, and others, which in turn affect all derived ones,
+     * such as alpha_fine_structure.
+     * This difference results in different numerical values for the material
+     * properties calculated by Celeritas.
+     */
+    auto material = mat_host_ptr.materials[1];
+
+    EXPECT_EQ(material.matter_state, MatterState::solid);
+    EXPECT_SOFT_EQ(material.temperature, 293.15);         // [K]
+    EXPECT_SOFT_EQ(material.density, 8.0000013682841882); // [g/cm^3]
+    EXPECT_SOFT_EQ(material.electron_density,
+                   2.2444324067595884e+24); // [1/cm^3]
+    EXPECT_SOFT_EQ(material.number_density,
+                   8.6993504137968536e+22);                  // [1/cm^3]
+    EXPECT_SOFT_EQ(material.rad_length, 1.7402560890562235); // [cm]
+    EXPECT_EQ(material.elements.size(), 3);
```

Comment:
This is a nit to pick, but the order of the arguments in the EXPECT macros matters a little bit (especially to soft equiv macros). In the case of a failure, the soft equiv error message will write ("expected ... but got ..."). In older versions of googletest `EXPECT_EQ` did this as well, but doesn't any more -- yet keeping a consistent expected/actual order across all our unit tests helps with readability.